CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
   This will make the cluster boot up correctly in IPv6 only environments, where IPv4 preference will break it due to lack of IPv4 addresses.
 * Improved the MirrorMaker2 example Grafana dashboard to set metric units and include chart descriptions.
 * The `ContinueReconciliationOnManualRollingUpdateFailure` feature gate moves to GA stage and is permanently enabled without the possibility to disable it.
+* Update OAuth library to 0.16.1 with removed support for KeycloakAuthorizer ACL delegation in Zookeeper mode.
```

Comment:
This is irrelevant as there is no way for anyone to use it?

Reply:
I usually add this information about the bump in the bridge CHANGELOG as well, why not here?

Reply:
I do not mind the info about 0.16.1 being used. But the `with removed support for KeycloakAuthorizer ACL delegation in Zookeeper mode` part seems confusing at best as it seems to be completely irrelevant to Strimzi Operators users.